lib/manageiq/environment.rb

```diff
@@ -24,8 +24,10 @@ def self.manageiq_plugin_update(plugin_root = nil, force_bundle_update: true)
       ensure_config_files
 
       unless ENV["CI"]
```

Comment:
On a separate topic, we have defined a separate ENV var for this step in the core bin/setup, _but_ I don't think it's available in the plugin's bin/setup.

https://github.com/ManageIQ/manageiq/blob/b6aef6613eba07e0d4c342eb8f185fc3d4bb35c8/bin/setup#L17

I'm going to do this in a followup, but it might be preferable to default that same variable in the methods above, and that would allow the caller to do something like `SKIP_DATABASE_SETUP=false` in a CI env if they want to have the development database setup anyway (for example in a cypress test suite)